### PR TITLE
Create CONTRIBUTING.md

### LINKED ISSUE #4: add contribution guidelines

### PR DESCRIPTION
**Pull request by rohitjmathew**
Fixes #4

### CODE REVIEW

**Comment by rohitjmathew** [comment]
@priyansh32 please review and add `hacktoberfest-approved` label to ensure this is eligible for Hacktoberfest

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+If you have suggestions for how this project could be improved, or want to report a bug, open an issue! We'd love all and any contributions. If you have questions, too, we'd love to hear them.
+
+We'd also love PRs. If you're thinking of a large PR, we advise opening up an issue first to talk about it, though! Look at the links below if you're not sure how to open a PR.
+
+## Submitting a pull request
+
+1. [Fork][fork] and clone the repository.
+1. Configure and install the dependencies: `npm install`.
+1. Make sure the tests pass on your machine: `npm test`, note: these tests also apply the linter, so there's no need to lint separately.
```
Comment (priyansh32) [inline CONTRIBUTING.md:19]: currently we do not have tests configured. Can you just change it to run `npm run lint` to lint code

**Review by priyansh32 — CHANGES_REQUESTED** [review]
Thanks for the PR @rohitjmathew 

Can you move this file to .github directory? There is already a empty CONTRIBUTING.md file in there. You can remove that.

File: CONTRIBUTING.md (hunk 1 above)
Comment (rohitjmathew) [reply CONTRIBUTING.md]: Done 👍

Comment (rohitjmathew) [reply CONTRIBUTING.md]: Done. Changed the command here but kept adding test cases in point 5 as that is a standard best practice and I would encourage doing it.

**Review by priyansh32 — APPROVED** [review]

**Comment by priyansh32** [comment]
Thanks for the contribution @rohitjmathew  😄

**Comment by rohitjmathew** [comment]
> Thanks for the contribution @rohitjmathew 😄

No worries. I was just searching for some quick hacktober contributions I could do before I go to sleep.

**Comment by rohitjmathew** [comment]
@priyansh32 could you add the label `hacktober` also. Looks like the contribution didn't get tracked.

**Comment by rohitjmathew** [comment]
Thanks @priyansh32 it got tracked now 👍

**Pull request merged**